packages/client/helpers/build.ts

```diff
@@ -55,7 +55,8 @@ const edgeRuntimeBuildConfig: BuildOptions = {
   define: {
     // that helps us to tree-shake unused things out
     NODE_CLIENT: 'false',
-    TARGET_ENGINE_TYPE: '"data-proxy"',
+    // tree shake the Library and Binary engines out
+    TARGET_ENGINE_TYPE: '"edge"',
```

Comment:
Just a change of name because there is no runtime called `data-proxy.js` anymore.